### PR TITLE
Adds loading different levels capability.

### PR DESCRIPTION
**Pull request by abishekvashok**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), #161 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]: #161 

### My steps to solve it,
Have included code to make it work.
Now it will load the next level once we reach the goal tile.

### Links

[http://rawgit.com/abishekvashok/labyrinth/master/index.html]

This was a collaboration by me (@abishekvashok), @samyakbvs and @YashKumarVerma

### CODE REVIEW

**Comment by abishekvashok** [comment]
Note: Game is not working due to #331

**Comment by arjunpandey2003** [comment]
Nice addtion guys

File: js/levels.js [hunk 1]
```diff
@@ -8,50 +8,39 @@
...
-    [desert.none, desert.right, desert.right, desert.right, desert.right, desert.none],
-    [desert.none, desert.top, desert.both, desert.both, desert.both, desert.both],
-    [desert.none, desert.top, PlayerStartsAt(desert.start), desert.both, desert.both, desert.top],
-    [desert.none, desert.top, desert.both, desert.both, desert.both, desert.top],
-    [desert.none, desert.top, desert.top, desert.both, desert.top, desert.top],
-    [desert.none, desert.top, desert.both, desert.both, desert.both, desert.top],
-    [NullTile, desert.none, desert.none, desert.none, desert.none, desert.none],
-  ]);
+function createTestLevel(level) {
+  if(level == 2) {
```
Comment (niccokunzmann) [inline js/levels.js]: Yes, this works. I would prefer an array. They do dispatch on numbers. That would be more object oriented and less procedural style.

**Comment by niccokunzmann** [comment]
I would like to merge this but I have merge conflicts.

**Comment by samyakbvs** [comment]
Thanks @arjunpandey2003

**Comment by YashKumarVerma** [comment]
@abishekvashok Conflicts to be solved

**Comment by YashKumarVerma** [comment]
@abishekvashok Conflicts to be solved

**Comment by abishekvashok** [comment]
@niccokunzmann @0x48piraj @YashKumarVerma done.

File: js/levels.js (hunk 1 above)
Comment (abishekvashok) [reply js/levels.js]: The array would be large @niccokunzmann. I can do it if u want, do you want me to?

File: js/tiles.js [hunk 2]
```diff
@@ -803,6 +805,7 @@ const door = {
 
 const forest = {
     both: OpenDoors,
+    levelCode: 2,
```
Comment (niccokunzmann) [inline js/tiles.js]: There are tiles and not levels. So I think, if we mix that up as it may confuse people that tiles can not be used across levels.

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**